README.md

```diff
@@ -638,6 +640,53 @@ handle dependencies and CLDR loading manually yourself.
   [Read more...](doc/error/e-unsupported.md)
 
 
+## Contribution
```

Comment:
Re-read docs and I see "Getting started". So, I am changing "Contribution" to "Contributing" as you suggest.